### ISSUE TITLE
Fail to build pyngp on windows - instant-ngp\dependencies\pybind11\include\pybind11\detail/type_caster_base.h(448): error : pointer to incomplete class type "_frame" is not allowed

### ISSUE THREAD

**Issue by elrobochanco** [NVlabs/instant-ngp#1043]
I'm trying to build on windows and everything works/completes except pyngp. When running cmake it identifies my install and development module:

```
(found suitable version "3.11.0", minimum required is "3.7") found components: Interpreter Development Development.Module Development.Embed
-- pybind11 v2.7.1
```

But once it is compiling python_api.cu I get:
```
instant-ngp\dependencies\pybind11\include\pybind11\detail/type_caster_base.h(448): error : pointer to incomplete class type "_frame" is not allowed [instant-ngp\build\pyngp.vcxproj]

instant-ngp\dependencies\pybind11\include\pybind11\detail/type_caster_base.h(448): error : type name is not allowed instant-ngp\build\pyngp.vcxproj]

instant-ngp\dependencies\pybind11\include\pybind11\detail/type_caster_base.h(448): error : expected an expression [instant-ngp\build\pyngp.vcxproj]

instant-ngp\dependencies\pybind11\include\pybind11\detail/type_caster_base.h(450): error : pointer to incomplete class type "_frame" is not allowed [instant-ngp\build\pyngp.vcxproj]

instant-ngp\dependencies\pybind11\include\pybind11\detail/type_caster_base.h(450): error : type name is not allowed [instant-ngp\build\pyngp.vcxproj]

instant-ngp\dependencies\pybind11\include\pybind11\detail/type_caster_base.h(450): error : expected an expression [instant-ngp\build\pyngp.vcxproj]

instant-ngp\dependencies\pybind11\include\pybind11\detail/type_caster_base.h(451): error : pointer to incomplete class type "_frame" is not allowed [iinstant-ngp\build\pyngp.vcxproj]

instant-ngp\dependencies\pybind11\include\pybind11/pybind11.h(2163): error : class "_ts" has no member "frame" [instant-ngp\build\pyngp.vcxproj]

instant-ngp\dependencies\pybind11\include\pybind11/pybind11.h(2164): error : pointer to incomplete class type "_frame" is not allowed [instant-ngp\build\pyngp.vcxproj]

instant-ngp\dependencies\pybind11\include\pybind11/pybind11.h(2165): error : pointer to incomplete class type "_frame" is not allowed [instant-ngp\build\pyngp.vcxproj]

instant-ngp\dependencies\pybind11\include\pybind11/pybind11.h(2168): error : pointer to incomplete class type "_frame" is not allowed [instant-ngp\build\pyngp.vcxproj]

instant-ngp\dependencies\pybind11\include\pybind11/pybind11.h(2168): error : pointer to incomplete class type "_frame" is not allowed [instant-ngp\build\pyngp.vcxproj]

  12 errors detected in the compilation of "instant-ngp/src/python_api.cu".
```

I'm not familiar with pybind, there is a small comment saying this won't work on PyPy, above the section but I just have python for windows. As everything else works I can run instant NGP, I just can't use any of the run.py commands such as exporting frames.
pybind11.h
![image](https://user-images.githubusercontent.com/33557736/198864272-d8b75c2d-8d4d-48f9-93e3-f102d49ed08b.png)
type_caster_base.h
![image](https://user-images.githubusercontent.com/33557736/198864291-6b61c741-4e44-4765-8ecb-5478f9207704.png)

Thanks for any guidance,  a bit out of my depth. Hope its something silly.

**Comment by SamuHell** [comment]
I'm getting this error as well, subscribing to this issue

**Comment by elrobochanco** [comment]
Slight progress, I changed my python install to 3.10 and reinstalled modules. Now it builds pyngp.cp310-win_amd64.pyd, but using it still fails to use it, as run.py says it can't import it:

`import pyngp as ngp # noqa
ImportError: DLL load failed while importing pyngp: The specified module could not be found.`

Which is odd as it's in the right folder, I have made sure its in the path, and I have tried other suggestions about re-naming it to just pyngp.pyd.

**Comment by elrobochanco** [comment]
Ok I have figured it out and past the python version it was a silly mistake on my side.

Python 3.11 changed frame objects so moving back to 3.10 allowed the file to be compiled correctly.
My follow-on issue launching through run.py was due to me copying an example command and not reversing the `\` orientation for windows. Even though I had done so for all other build commands.

So to summarize:

1.  Use Python 3.10 not 3.11
2. `python scripts/run.py --mode nerf --scene data/nerf/fox..... `should be `python scripts\run.py --mode nerf --scene data\nerf\fox.....` on windows
